x-pack/plugins/apm/public/new-platform/plugin.tsx

```diff
@@ -10,8 +10,9 @@ import { Provider } from 'react-redux';
 import { Router } from 'react-router-dom';
 import { CoreSetup } from 'src/core/public';
 import { Main } from '../components/app/Main';
-import { history } from '../components/shared/Links/url_helpers';
+import { history } from '../utils/history';
 import { LocationProvider } from '../context/LocationContext';
+import { UrlParamsProvider } from '../context/UrlParamsContext';
```

Comment:
I like how you've organised this under a context folder and with consistent `Provider` suffix naming.
Would you mind doing the same for `LicenseCheck` and `GlobalFetchIndicator` (if it makes sense)

Reply:
FYI I didn't get to this in this ticket